screen-onboarding/src/main/java/com/ivy/onboarding/viewmodel/OnboardingViewModel.kt

```diff
@@ -62,28 +65,30 @@ class OnboardingViewModel @Inject constructor(
     transactionReminderLogic: TransactionReminderLogic,
     preloadDataLogic: PreloadDataLogic,
     logoutLogic: LogoutLogic,
-) : ViewModel() {
-
-    private val _state = MutableLiveData(OnboardingState.SPLASH)
-    val state = _state.asLiveData()
-
-    private val _currency = MutableLiveData<IvyCurrency>()
-    val currency = _currency.asLiveData()
-
-    private val _opGoogleSignIn = MutableLiveData<OpResult<Unit>?>()
-    val opGoogleSignIn = _opGoogleSignIn.asLiveData()
-
-    private val _accounts = MutableLiveData<ImmutableList<AccountBalance>>()
-    val accounts = _accounts.asLiveData()
-
-    private val _accountSuggestions = MutableLiveData<ImmutableList<CreateAccountData>>()
-    val accountSuggestions = _accountSuggestions.asLiveData()
-
-    private val _categories = MutableLiveData<ImmutableList<Category>>()
-    val categories = _categories.asLiveData()
-
-    private val _categorySuggestions = MutableLiveData<ImmutableList<CreateCategoryData>>()
-    val categorySuggestions = _categorySuggestions.asLiveData()
+) : ComposeViewModel<OnboardingDetailState, OnboardingEvent>() {
+
+    private val _state = mutableStateOf(OnboardingState.SPLASH)
```

Comment:
Remove the `_` from the name

Reply:
This applies for all states

Reply:
Hey @ILIYANGERMANOV, I have used `_` only for the state variables private to the viewmodel, so as to prevent confusion and maintain clarity as the constructor parameters of the exposed uiState have those without `_`.